### PR TITLE
Add App Center icon

### LINKED ISSUE #7280: Request: App Center
### Brand Name

App Center

### Website

https://appcenter.ms

### Popularity Metric

Similarweb rank: 14,497
https://www.similarweb.com/website/appcenter.ms/#overview

### Official Resources for Icon and Color

ico: https://appcenter.ms/favicon.ico
color: #cb2e62

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by zrfisaac**
appcenter

**Issue:** Closes #7280

**Similarweb rank:** [13,847](https://appcenter.ms/)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by zrfisaac** [comment]
I don't know what is wrong with this SVG, anybody have an idea ?

**Comment by PratyushJayachandran** [comment]
> I don't know what is wrong with this SVG, anybody have an idea ?

Did you convert the PNG on the website front page to the SVG on this pull request? If you used some other official source, can you provide that instead?
Will help in reviewing and solving the linter errors :)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -775,6 +775,11 @@
             "hex": "0F2346",
             "source": "https://www.appannie.com/en/about/press/"
         },
+        {
+            "title": "App Center",
+            "hex": "CB2E62",
+            "source": "https://visualstudio.microsoft.com/app-center/"
```
Comment (adamrusted) [inline _data/simple-icons.json:781]:
```suggestion
            "source": "https://learn.microsoft.com/en-us/visualstudio/subscriptions"
```

SVG version of the logo available here.

File: icons/appcenter.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>App Center</title><path d="M18 0v20.6L0 18l18 6 6-2.5v-19zM6 4 2 7.5V9l2.3 1.5L2 12v1.5L6 17l2.2-.5v-3.6l3.8 2.6 3-1v-8l-3-1-3.7 2.6V4.5zm0 2v3.5L3.2 8.1zm6 2.5v4l-3.8-2zm-6 3V15l-2.8-2.1z"/></svg>
```
Comment (adamrusted) [inline icons/appcenter.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Visual Studio App Center</title><path d="M18.133 3.307 11.2 7.92V1.133h-.16L7.373.007 0 6.053v2.834l4.773 3.106L0 15.1v2.833l7.373 6.06 3.827-1.14v-6.786l6.933 4.613L24 18.873V5.113l-5.867-1.806Zm-15.84 4L6.96 4.04v5.747l-4.667-2.48ZM6.96 14.26v5.747L2.293 16.74l4.667-2.48Zm11.707 1.48-7.547-3.747 7.547-3.746v7.493Z"/></svg>
```

**Comment by adamrusted** [comment]
I'm going to close this PR, and open a new one - I don't appear to have adequate permissions on your branch to make the right changes. If you're able to rectify the issue (remove the newline in the SVG) we can re-open this PR.

**Pull request closed without merging**